martian/src/stage.rs

```diff
@@ -117,14 +117,16 @@ impl<T: MartianFileType> MartianMakePath for T {
 ///
 /// Memory/ thread request can be negative in matrian. See
 /// [http://martian-lang.org/advanced-features/#resource-consumption](http://martian-lang.org/advanced-features/#resource-consumption)
-#[derive(Debug, Serialize, Deserialize, Copy, Clone, Default)]
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
 pub struct Resource {
     #[serde(rename = "__mem_gb")]
     mem_gb: Option<isize>,
     #[serde(rename = "__threads")]
     threads: Option<isize>,
     #[serde(rename = "__vmem_gb")]
     vmem_gb: Option<isize>,
+    #[serde(rename = "__special")]
+    special: Option<String>,
```

Comment:
This cannot be specified in the `using` section of the mro right?

Reply:
It can be specified

Reply:
Okay, to support it in the using section, we would need to update https://github.com/martian-lang/martian-rust/blob/7028107cbf14e751b98b7787594ddb0fa92ae2b1/martian/src/mro.rs#L657 and https://github.com/martian-lang/martian-rust/blob/7028107cbf14e751b98b7787594ddb0fa92ae2b1/martian-derive/src/lib.rs#L392. This is only needed if we want to specify `#[make_mro(special="...")]`

Reply:
This should really be
```suggestion
    special: Option<&' static str>,
```
There is no plausible use case for this being something other than a compile-time constant.

Reply:
(yes, I know this poses problems for deserialization but I can't think of a plausible case where you need to deserialize it either)